sycl/doc/syclcompat/README.md

```diff
@@ -947,6 +947,42 @@ public:
 };
 ```
 
+### ptr_to_int
```

Comment:
Sorry to be a pain, but this is currently sandwiched between 2 'devicey' sections. I'd like to suggest moving it up to just above `### Device Information` on L858, and making it a subsection of memory_operations (`#### ptr_to_int`).

Reply:
Yeah good point, thanks